var/spack/repos/builtin/packages/py-pager/package.py

```diff
@@ -10,7 +10,7 @@ class PyPager(PythonPackage):
     """Python module that pages output to the screen,
        reads keys and console dimensions without executing external utils."""
 
-    homepage = "http://bitbucket.org/techtonik/python-pager"
+    homepage = "https://pypi.org/project/pager/"
```

Comment:
You can remove this line, it's the default set by `pypi`

Reply:
This should be fixed.